src/components/cursor.js

```diff
@@ -59,7 +59,7 @@ module.exports.Component = registerComponent('cursor', {
     // Debounce.
     this.updateCanvasBounds = utils.debounce(function updateCanvasBounds () {
       self.canvasBounds = self.el.sceneEl.canvas.getBoundingClientRect();
-    }, 200);
+    }, 1000);
```

Comment:
Is this necessary?

Reply:
Not really, but really don't need to check window resize that often.